drive/types.bal

```diff
@@ -123,76 +123,76 @@ public type FileMetadata record {
+# + sharedWithMeTime - The time at which the file was shared with the user, if applicable (RFC 3339 date-time).
+#                     Not populated for items in shared drives
+# + webViewLink - A link for opening the file in a relevant Google editor or viewer in a browser
+# + ownedByMe - Whether the user owns the file. Not populated for items in shared drives
 # + explicitlyTrashed - Whether the file has been explicitly trashed, 
-#                       as opposed to recursively trashed from a parent folder.  
-# + trashedTime - The time that the item was trashed (RFC 3339 date-time). Only populated for items in shared drives.  
-# + viewedByMe - Whether the file has been viewed by this user.  
+#                       as opposed to recursively trashed from a parent folder
+# + trashedTime - The time that the item was trashed (RFC 3339 date-time). Only populated for items in shared drives  
+# + viewedByMe - Whether the file has been viewed by this user
 # + driveId - ID of the shared drive the file resides in. Only populated for items in shared drives  
 # + size - The size of the file's content in bytes. 
-#          This is applicable to binary files in Google Drive and Google Docs files.    
+#          This is applicable to binary files in Google Drive and Google Docs files    
 # + spaces - The list of spaces which contain the file. The currently supported values are 'drive', 
-#            'appDataFolder' and 'photos'.  
-# + imageMediaMetadata - Additional metadata about image media, if available. 
+#            'appDataFolder' and 'photos'
+# + imageMediaMetadata - Additional metadata about image media, if available
 # + parents - The IDs of the parent folders which contain the file. If not specified as part of a create request, 
 #             the file will be placed directly in the user's My Drive folder. 
 #             If not specified as part of a copy request, 
-#             the file will inherit any discoverable parents of the source file. 
+#             the file will inherit any discoverable parents of the source file
 #             Update requests must use the addParents and removeParents parameters to modify the parents list. 
 # + headRevisionId - The ID of the file's head revision. This is currently only available for files with binary content 
-#                    in Google Drive.  
-# + modifiedByMeTime - The last time the file was modified by the user (RFC 3339 date-time). 
-# + modifiedByMe - Whether the file has been modified by this user.
-# + shared - Whether the file has been shared. Not populated for items in shared drives.  
+#                    in Google Drive
+# + modifiedByMeTime - The last time the file was modified by the user (RFC 3339 date-time)
+# + modifiedByMe - Whether the file has been modified by this user
+# + shared - Whether the file has been shared. Not populated for items in shared drives 
 # + hasAugmentedPermissions - Whether there are permissions directly on this file. 
```

Comment:
Suggestion: "Indicates whether there are permissions directly on this file. This is displayed only. for items in shared drive""

Reply:
Hi,
We have used the same content from [1] for parameter descriptions
[1]https://developers.google.com/drive/api/v3/reference/files
Thanks